descartes_core/include/descartes_core/cart_trajectory_pt.h

```diff
@@ -70,8 +70,8 @@ struct ToleranceBase
     #param tol Total tolerance zone (assumed symetric about nominal)
     */
   template<typename T>
-  static T createSymmetric(const double x, const double y, const double z,
-                                          const double tol)
+  static T createSymmetric(const double& x, const double& y, const double& z,
```

Comment:
I thought that passing by reference was the original intent since in the original version of this function the 'const' attribute proceeded all the arguments.  When passing by value then 'const' has no effect on the argument as far as I know.